src/components/GitPanel.tsx

```diff
@@ -504,7 +516,9 @@ export class GitPanel extends React.Component<IGitPanelProps, IGitPanelState> {
         }
         this._previousRepoPath = path;
       } catch (error) {
-        throw new Error('Failed to set your identity. ' + error.message);
+        throw new Error(
+          this.props.trans.__('Failed to set your identity. ') + error.message
```

Comment:
We might want to bundle the whole thing together, in case on other languages the order is not the expected

```
this.props.trans.__('Failed to set your identity.  %1', error.message)
```